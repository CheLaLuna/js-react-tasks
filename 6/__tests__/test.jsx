import React from 'react';
import { render } from '@testing-library/react';
import Alert from '../src/Alert.jsx';

test('Alert 1', () => {
  const params = {
    type: 'danger',
    text: 'text 1',
  };
  const { asFragment } = render(<Alert {...params} />);
  expect(asFragment()).toMatchSnapshot();
});

test('Alert 2', () => {
  const params = {
    type: 'dark',
    text: 'text 2',
  };
  const { asFragment } = render(<Alert {...params} />);
  expect(asFragment()).toMatchSnapshot();
});
