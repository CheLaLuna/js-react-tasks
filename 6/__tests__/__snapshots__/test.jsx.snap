// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert 1 1`] = `
<DocumentFragment>
  <div
    class="alert alert-danger"
    role="alert"
  >
    text 1
  </div>
</DocumentFragment>
`;

exports[`Alert 2 1`] = `
<DocumentFragment>
  <div
    class="alert alert-dark"
    role="alert"
  >
    text 2
  </div>
</DocumentFragment>
`;
